tests/test_rooms/test_room_analytics_with_no_meetings.py

```diff
@@ -0,0 +1,24 @@
+import json
+from unittest.mock import patch
+
+from tests.base import BaseTestCase, CommonTestCases
+from fixtures.room.most_booked_room_least_booked_rooms_fixtures import (
+    test_for_division_error
+)
+
+with open('events.json', 'r') as outfile:
```

Comment:
instead of repeating this context manager for opening the json file everywhere, we can have it in a function and import it to any file where we have to use it.

Reply:
@michael-basweti, sounds even better. Let me work on it.